wml_test_schedule

```diff
@@ -397,6 +396,10 @@
 0 unknown_scenario_false_positives
 0 unknown_scenario_interpolated
 9 unknown_scenario_1_0
+9 unknown_scenario_1_0_last
+9 unknown_scenario_1_1
+9 unknown_scenario_2_0
+9 unknown_scenario_1_1_last_null
```

Comment:
While these can be added, they're slow (as broke-strict tests have to be run individually), and the automatic testing mechanism won't see the difference between them. They show error dialogs that are intended to be read by end-users (players, not just WML authors), and the different tests show slightly different error messages for manual testing when the tester is reading what's displayed on the UI.

However, if the aim is to have a scripted check that all tests defined in data/test/scenarios are run then I understand adding them.

Reply:
My thought was "they're there and have a valid end state they're testing, so why not?". There are a couple test scenarios that don't (they're solely meant for someone to run manually) which I'm not sure whether they should be in this directory and where to put them otherwise, but that's a separate thing.